ios/brave-ios/Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -75,6 +75,69 @@ public struct AssetGroupViewModel: WalletAssetGroupViewModel, Identifiable, Equa
       return partialResult + assetValue
     }
   }
+
+  /// Sort by the group's total fiat/value
+  static func sorted(
```

Comment:
I think we should be able to share some of this duplicated code with these new `AssetGroupViewModel` sort functions with the existing `AssetViewModel` sort functions? Likely extension(s) / sort function for `AccountInfo` & `NetworkInfo` would clean these up, maybe in `BraveWalletExtensions.swift`.